infrastructure/cdn-in-a-box/traffic_ops/Dockerfile

```diff
@@ -37,22 +37,14 @@ RUN set -o nounset -o errexit && \
 	if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
 		use_repo=''; \
 		enable_repo=''; \
-		llvm_version=5.0; \
-		# needed for llvm-toolset-7-clang, which is needed for postgresql13-devel-13.2-1PGDG, required by TO rpm
-		dnf -y install gcc centos-release-scl-rh; \
 	else \
 		use_repo='--repo=pgdg13'; \
 		enable_repo='--enablerepo=powertools'; \
-		llvm_version=''; \
 	fi; \
 	dnf -y install "https://download.postgresql.org/pub/repos/yum/reporpms/EL-${RHEL_VERSION%%.*}-x86_64/pgdg-redhat-repo-latest.noarch.rpm"; \
 	dnf -y install epel-release; \
-	dnf -y install \
-		# libicu is required by postgresql13
-		libicu \
-		# libicu-devel, clang-devel, and llvm-devel are required by postgresql13-devel
-		libicu-devel clang-devel llvm${llvm_version}-devel; \
-	dnf -y $use_repo -- install postgresql13 postgresql13-devel; \
+	dnf -y install --nobest libicu; \
```

Comment:
This line could use a comment above explaining why `--nobest` is used.

Reply:
_**I**_ could use a comment explaining why `--nobest` is used. I don't know why it used to work without that, why it doesn't anymore, or what "best" means to `yum`/`dnf`

Reply:
It installs for me without `--nobest`

Reply:
Works for me now too